technical-operations/middleware/launchers/Cloudcashbox.md

```diff
@@ -0,0 +1,23 @@
+---
+slug: /posdealers/technical-operations/middleware/launchers/CloudCashbox
+title: CloudCashbox
+---
+
+# CloudCashbox
+
+:::info summary
+
+After reading this, you can explain what the _CloudCashbox_ solution is, how it is used to host the Middleware.
+
+:::
+
+## Introduction
+The CloudCashbox is a fully hosted SaaS version of fiskaltrust's Middleware, offering a centralized and reliable fiscalization solution. Hosted on Microsoft Azure in Germany, it simplifies the complexity of fiscal compliance by eliminating the need for local software configuration and installation. Users can easily connect to the Middleware using an access token and URL, without the need to manage TSEs.
+
+###Key features include:
```

Comment:
missing space between # and Key

Reply:
Thanks for the review :) 
Change made with commit b1008d2